core/panic.c

```diff
@@ -28,6 +28,10 @@
 #include "panic.h"
 #include "arch/panic_arch.h"
 
+#if DEVELHELP && defined MODULE_PS
```

Comment:
`-DDEVELHELP` on the command line is the same as `-DDEVELHELP=1`, at least on GCC and Clang